apps/condo/domains/common/components/IFrame/IFrameWrapper.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react'
+import { sendLoadedStatus, sendRequirementRequest } from '@condo/domains/common/utils/iframe.utils'
+import { useAuth } from '@core/next/auth'
+import { useOrganization } from '@core/next/organization'
+import getConfig from 'next/config'
+
+interface IFrameWrapperProps {
+    withOrganization?: boolean
+    withUser?: boolean
+    parentOrigin?: string
+}
+
+const { publicRuntimeConfig: { serverUrl } } = getConfig()
+
+export const IFrameWrapper: React.FC<IFrameWrapperProps> = (props) => {
+    const { withUser, withOrganization, parentOrigin: propParentOrigin } = props
+    const parentOrigin = propParentOrigin ? propParentOrigin : serverUrl
+    const parentType = typeof parent
+    const { isAuthenticated } = useAuth()
+    const { organization } = useOrganization()
+
+    useEffect(() => {
```

Comment:
I think all of 3 effects can be combined in one

Reply:
They have different dependencies and executes request to parent window, that's why some of send will be called multiple times. That's why I decided to separate it 👀